templates/experiments/single_experiment_home.html

```diff
@@ -212,54 +212,215 @@ <h3 class="font-bold text-lg">Parent Experiments:
+                            </template>
+                          <!-- String Input for string type -->
+                            <template x-if="filterData.columns[filter.column]?.type === 'string'">
+                              <input
+                                type="text"
+                                x-model="filter.value"
+                                @input="triggerFilterChange()"
+                                class="block w-40 py-2 px-3 border border-gray-200 rounded-lg bg-white text-sm"
+                                placeholder="Enter value..."
+                              >
+                            </template>
+                          <!-- Timestamp Input -->
+                            <template x-if="filterData.columns[filter.column]?.type === 'timestamp'">
+                              <input
+                                type="date"
+                                x-model="filter.value"
+                                @change="triggerFilterChange()"
+                                class="block w-40 py-2 px-3 border border-gray-200 rounded-lg bg-white text-sm"
+                              >
+                            </template>
+                          <!-- Regular select dropdown for choice type with non-multiselect operators -->
+                            <template x-if="filterData.columns[filter.column]?.type === 'choice' && !['any of', 'all of'].includes(filter.operator)">
+                              <select
+                                x-model="filter.value"
+                                @change="triggerFilterChange()"
+                                class="block w-40 py-2 px-3 border border-gray-200 rounded-lg bg-white text-sm"
+                              >
+                                <option value="">Select value</option>
+                                <template x-for="option in filterData.columns[filter.column]?.options" :key="option">
+                                  <option :value="option" x-text="option"></option>
+                                </template>
+                              </select>
+                            </template>
+                          </div>
+                        </template>
+                      <!-- Remove Filter Button-->
+                        <button
+                          @click="removeFilter(index)"
+                          class="p-2 text-gray-400 hover:text-gray-600 h-10 w-10 flex items-center justify-center"
+                          aria-label="Remove filter"
+                        >
+                          <span class="text-lg">×</span>
+                        </button>
+                      </div>
+                    </template>
+                  <!-- Add Filter Button -->
+                    <button
+                      @click="addFilter()"
+                      class="flex items-center gap-2 px-4 py-2 mt-3 text-gray-700 border border-gray-200 rounded-lg hover:bg-gray-50 text-sm"
```

Comment:
For consistency with the rest of the UI it might be better to use the DaisyUI button classes: https://v4.daisyui.com/components/button/

Possibly similar comment for other buttons. Also not a blocker

Reply:
changed for all but one that was giving me a particularly hard time